### PR TITLE
Added Excuser API in Entertainment Category

### PR DESCRIPTION
**Pull request by onlykingKD**
- [x] My submission is formatted according to the guidelines in the [contributing guide](/CONTRIBUTING.md)
- [x] My addition is ordered alphabetically
- [x] My submission has a useful description
- [x] The description does not have more than 100 characters
- [x] The description does not end with punctuation
- [x] Each table column is padded with one space on either side
- [x] I have searched the repository for any relevant issues or pull requests
- [x] Any category I am creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -654,8 +654,10 @@ API | Description | Auth | HTTPS | CORS |
 | [Meme Maker](https://mememaker.github.io/API/) | REST API for create your own meme | No | Yes | Unknown |
 | [NaMoMemes](https://github.com/theIYD/NaMoMemes) | Memes on Narendra Modi | No | Yes | Unknown |
 | [Random Useless Facts](https://uselessfacts.jsph.pl/) | Get useless, but true facts | No | Yes | Unknown |
+| [Excuser](https://excuser.herokuapp.com/) | Get random excuses for various situations | No | Yes | Unknown |
 | [Yo Momma Jokes](https://github.com/beanboi7/yomomma-apiv2) | REST API for Yo Momma Jokes | No | Yes | Unknown |
 
+
```
Comment (matheusfelipeog) [inline README.md:660]: Remove this new line, please:

```suggestion
```

**Review by matheusfelipeog — CHANGES_REQUESTED** [review]
Thanks for your contribution @onlykingKD ;)

Can you fix the alphabetical order, please? This API must be between chucknorris.io and Imgflip.

File: README.md (hunk 1 above)
Comment (onlykingKD) [reply README.md]: Done

**Review by matheusfelipeog — APPROVED** [review]
Now it's Ok ;)

Thanks for the update @onlykingKD <3

**Pull request merged**